nucleus/admin/cli/src/main/java/com/sun/enterprise/admin/cli/CLICommand.java

```diff
@@ -1184,12 +1204,14 @@ private String passwordName(ParamModel opt) {
      * @param prompt
      * @return 
      */
-    protected char[] readPassword(String prompt) {
+    protected char[] readPassword(String prompt) throws IOException {
```

Comment:
Couldn't you catch the IOException here and throw some RumtimeException. Like IllegalStateException or something.

Reply:
Upon review it's actually meant to just return null, so I'm going to change it to do that.